### PR TITLE
[Bugfix] [FE] Bug - user gets 401 Unauthorized error first time they enter the app (IMB-178)

### PR DESCRIPTION
**Pull request by DonAlexandro**
## Overview
This PR will fix `401 Unauthorized` on initial login

## Test cases

1. Sign in with Google and use an email that doesn't exist in our internal DB
2. Verify that `401 Unauthorized` error doesn't appear when you log in

## Issue

closes [IMB-178](https://www.notion.so/FE-Bug-user-gets-401-Unauthorized-error-first-time-they-enter-the-app-699cdf1a1fa744e3b332b48c415b8f22?pvs=4)

### CODE REVIEW

File: mobile/android/app/google-services.json [hunk 1]
```diff
@@ -56,51 +56,7 @@
           ]
         }
       }
-    },
```
Comment (vladyslav-panchenko280) [inline mobile/android/app/google-services.json:59]: Why we delete it? (non-blocked)

File: mobile/ios/Runner/GoogleService-Info.plist [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	<key>REVERSED_CLIENT_ID</key>
 	<string>com.googleusercontent.apps.222626452683-823iovqtt4e9aukg6rl8bkr6c34oj7r5</string>
 	<key>ANDROID_CLIENT_ID</key>
-	<string>222626452683-6e62oh12qg57dg31226f5amnldeftm5d.apps.googleusercontent.com</string>
```
Comment (vladyslav-panchenko280) [inline mobile/ios/Runner/GoogleService-Info.plist:10]: What is it? (non-blocked)

**Review by vladyslav-panchenko280 — APPROVED** [review]

File: mobile/android/app/google-services.json (hunk 1 above)
Comment (DonAlexandro) [reply mobile/android/app/google-services.json:59]: I've just reinitialized firebase for previous version of flutter

File: mobile/ios/Runner/GoogleService-Info.plist (hunk 2 above)
Comment (DonAlexandro) [reply mobile/ios/Runner/GoogleService-Info.plist:10]: same as above

**Pull request merged**